tools/cosmovisor/testdata/download/cosmovisor/genesis/bin/autod

```diff
@@ -6,7 +6,7 @@ echo 'ERROR: UPGRADE "chain2" NEEDED at height: 49: zip_binary'
 
 # create upgrade info
 # this info contains directly information about binaries (in chain2->chain3 update we test with info containing a link to the file with an address for the new chain binary)
-echo '{"name":"chain2","height":49,"info":"{\"binaries\":{\"linux/amd64\":\"https://github.com/cosmos/cosmos-sdk/raw/main/tools/cosmovisor/testdata/repo/chain2-zip_bin/autod.zip?checksum=sha256:fd20f3163e5ab2ab430c0c2936de82fcf5f100a8497d7f121b9f34a1a4b96683\"}}"}' >$3
+echo '{"name":"chain2","height":49,"info":"{\"binaries\":{\"linux/amd64\":\"https://github.com/cosmos/cosmos-sdk/raw/main/tools/cosmovisor/testdata/repo/chain2-zip_bin/autod.zip?checksum=sha256:13767eb0b57bf51a0f43d49f6277d5df97d4dec672dc39822d23a82fb8e70a7b\"}}"}' >$3
```

Comment:
So, here I leave it to main, as it will pass as soon as this PR is merged. Otherwise, I need to select a commit, but I think it gets delete when this branch gets deleted.
I will verify here that everything goes well.

Reply:
Passed, as promised https://github.com/cosmos/cosmos-sdk/actions/runs/4079143992/jobs/7030203497 :D